trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -248,15 +335,27 @@ def generate_score_and_reasons(
             return score, reasons
 
         else:
-            score = mod_generated_utils.re_0_10_rating(response) / normalize
+            score = (
+                mod_generated_utils.re_configured_rating(
+                    response,
+                    min_score_val=min_score_val,
+                    max_score_val=max_score_val,
+                )
+                - min_score_val
+            ) / (max_score_val - min_score_val)
             warnings.warn(
                 "No supporting evidence provided. Returning score only.",
                 UserWarning,
             )
             return score, {}
 
     def context_relevance(
-        self, question: str, context: str, temperature: float = 0.0
+        self,
+        question: str,
+        context: str,
+        criteria: str = "",
+        output_space: str = "",
```

Comment:
Why use the output space string here instead of min/max value?

Reply:
updated in https://github.com/truera/trulens/pull/1282/commits/9ea986f25c513c2e2ef696ec078d965ed544a1a3